Sources/hspp/CardSets/CoreCardsGen.cpp

```diff
@@ -118,7 +119,19 @@ void CoreCardsGen::AddPriestNonCollect(std::map<std::string, Power>& cards)
 
 void CoreCardsGen::AddRogue(std::map<std::string, Power>& cards)
 {
-    (void)cards;
+    Power power;
+
+	// ------------------------------------------ SPELL - ROGUE
```

Comment:
please, align comment indentation. thx!

Reply:
ah.. okay =)
i checked it and find tab and white space error
thx

Reply:
Fixed.